tests/spread/plugins/rust/snaps/rust-workspace-fib/fib/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "fib"
+version = "0.1.0"
+authors = ["Chris Patterson <chris.patterson@canonical.com>"]
+edition = "2018"
```

Comment:
funny, you are behind the times.

Reply:
lol, I think 2015 and 2018 are the valid rust editions :wink: 2021 might get some love, so we'll have to revisit this then!

Reply:
Ha, for some reason I thought this was the "edition" for the package and not rust itself